var/spack/repos/builtin/packages/qgis/package.py

```diff
@@ -97,10 +98,11 @@ class Qgis(CMakePackage):
     # optionals
     depends_on('postgresql@8:', when='+postgresql')  # for PostGIS support
     depends_on('gsl', when='+georeferencer')  # for georeferencer
-    depends_on('grass@7.0.0', type=('build', 'link', 'run'), when='+grass7')  # for georeferencer
+    # grass@7.8.1 is the first version that supports proj@6
+    depends_on('grass@7.8.1:7.999.999', type=('build', 'link', 'run'), when='+grass7')  # for georeferencer
 
     # the below dependencies are shown in cmake config
-    depends_on('hdf5')
+#    depends_on('hdf5')
```

Comment:
If this package directly links to hdf5, I would still keep this dependency. Explicit is better than implicit. In the past, we've considered removing transitive dependencies from the compiler wrappers, but they broke a lot of packages, so we undid that change.

Reply:
It breaks the concretizer.  `Spack spec qgis` results in a spectacular error.

Reply:
Ugh, can't wait for that new concretizer...

Reply:
Until then, do you think we could do what's needed to ensure that things concretize?  Having specs that simply don't concretize out of the box is bad PR, IMHO.

Reply:
Yeah, I agree, this is fine.